packages/seacas/libraries/ioss/src/Ioss_DatabaseIO.C

```diff
@@ -1400,8 +1408,8 @@ namespace Ioss {
         }
 
         size_t block_data_offset = offset[i] * field_byte_size;
-        auto   retval =
-            get_field_internal(entity, field, (char *)data + block_data_offset, block_data_size);
+
+        auto retval = get_field_internal(entity, field, (char*)data + block_data_offset, block_data_size);
```

Comment:
Is the `block_data_size` accounting for offsetting into the `data` by `block_data_offset`.  That is, does `block_data_size` have `block_data_offset` subtracted from it somewhere before this use.

Reply:
Yes, that is correct